Source/WebCore/style/PseudoElementRequest.h

```diff
@@ -50,6 +51,13 @@ class PseudoElementRequest {
 
     // highlight name for ::highlight or view transition name for view transition pseudo elements.
     AtomString pseudoElementNameArgument;
+
+    bool styleIsCacheable()
+    {
+        return !scrollbarState && pseudoElementNameArgument.isNull();
+    }
+
+    static PseudoElementRequest NoPseudoElement = { PseudoId::None };
```

Comment:
Why is it `static` ?

Reply:
It would be wasteful to re-define this for every instance of PseudoElementRequest, this was meant to be the default value for PseudoElementRequest (though it doesn't look like this compiles)